project-base/storefront/components/Blocks/BlogPreview/BlogPreview.tsx

```diff
@@ -46,15 +49,19 @@ export const BlogPreview: FC = () => {
                 )}
             </div>
 
-            <div className="flex flex-col gap-16 vl:flex-row  vl:justify-between">
-                {!!blogMainItems && (
-                    <div className="flex flex-1 gap-6 vl:gap-16">
-                        <BlogPreviewMain articles={blogMainItems} />
-                    </div>
-                )}
+            {!!(fetchingArticles || fetchingBlogUrl) && <SkeletonMagazine />}
 
-                {!!blogSideItems && <BlogPreviewSide articles={blogSideItems} />}
-            </div>
+            {!fetchingArticles && !fetchingBlogUrl && !!(blogMainItems || blogSideItems) && (
```

Comment:
Actually nope. This is not conditional rendering where it is either this or that. If I remove this condition it will display skeleton and content during fetching.